plugins/feeds/public/otx_alienvault.py

```diff
@@ -88,22 +85,27 @@ def analyze(self, item):
 
                 obs.tag(tags)
                 obs.add_context(self.name, context)
-
+                investigation.link_to(obs, "Observed", "OTXAlienVault")
             elif type_ind in entity.EntityType:
-                entity.Entity(
+                ent = entity.Entity(
                     name=otx_indic["indicator"],
                     type=self._TYPE_MAPPING.get(otx_indic["type"]),
                 ).save()
-
+                investigation.link_to(ent, "Observed", "OTXAlienVault")
             elif type_ind in indicator.IndicatorType:
                 if type_ind == indicator.IndicatorType.yara:
-                    indicator.Indicator(
+                    ind_obj = indicator.Indicator(
                         name=f"YARA_{otx_indic['indicator']}",
```

Comment:
`name` here is the "human readable" name, so I'd avoid having underscores in the name. What is the typical content of `otx_indic['indicator']`? could we just use this?

Reply:
![image](https://github.com/yeti-platform/yeti/assets/1478484/0f311e63-a988-4e3b-9705-26588039d35d)

It's the hash oh the yara rule. I've just that

Reply:
Could we maybe parse and extract the name and description from the Yara rule and populate the `name` and `description` attributes of the object?